ajax/libs/hyphenopoly/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "hyphenopoly",
+  "description": "Hyphenation for node and Polyfill for client-side hyphenation.",
+  "keywords": [
+    "hyphenation",
+    "html",
+    "polyfill",
+    "hyphens",
+    "hyphen",
+    "soft",
+    "hyphenate",
+    "JavaScript"
+  ],
+  "homepage": "https://mnater.github.io/Hyphenopoly/",
+  "license": "MIT",
+  "author": {
+    "name": "Mathias Nater",
+    "email": "mathiasnater@gmail.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mnater/Hyphenopoly"
+  },
+  "filename": "Hyphenopoly.js",
+  "npmName": "hyphenopoly",
+  "npmFileMap": [
+    {
+      "basePath": "min",
+      "files": [
+        "**/*"
```

Comment:
I don't think we support file extention like `hpb` and `wasm`, are they required to this library? If so, we'd need to contact Cloudflare to evaluate of supporting them.

cc @MattIPv4

Reply:
@PeterDaveHello `wasm` has been requested to be added but has not yet been added (see #13145 cc @dknecht) and, as far as I know, `hpb` is not an extension in the whitelist.

Reply:
From the [project readme](https://github.com/mnater/Hyphenopoly#readme), the `hpb` files contain hyphenation patterns in a binary format. `hyphenEngine.wasm` is for "creating pattern trie and finding hyphenation points". I believe both are required.

(There is also a `hyphenEngine.asm.js` for browsers that don't support wasm, but I believe the library only does a wasm feature test to determine which file to load, i.e. `hyphenEngine.asm.js` isn't a fallback if loading `hyphenEngine.wasm` failed.)

Reply:
@simon-says could you help with getting `hpb` & `wasm` added to the whitelist?

Reply:
ping @simon-says @MattIPv4 - any progress on this?